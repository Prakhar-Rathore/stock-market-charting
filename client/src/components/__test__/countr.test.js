import React from "react";
import ReactDOM from "react-dom/client";
import Countr from "./../countr";
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { act } from 'react-dom/test-utils';

it("renders without crashing", async() => {
  const root = ReactDOM.createRoot(document.createElement("div"));
  await act( async () => render(<Countr/>));
});

it("renders correctly", () => {
  render(<Countr/>)
})

it("matches snapshot", () => {
  const tree = renderer.create(<Countr/>).toJSON();
  expect(tree).toMatchSnapshot();
})