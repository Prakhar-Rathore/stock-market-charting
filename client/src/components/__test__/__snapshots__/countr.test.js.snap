// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    className="MuiFormControl-root css-7ucr63-MuiFormControl-root"
  >
    <label
      className="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
      data-shrink={false}
    >
       Country
    </label>
    <div
      className="MuiFilledInput-root MuiFilledInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl  css-67qocj-MuiInputBase-root-MuiFilledInput-root-MuiSelect-root"
      onClick={[Function]}
      required={true}
      style={
        Object {
          "width": "290px",
        }
      }
    >
      <div
        aria-expanded="false"
        aria-haspopup="listbox"
        aria-labelledby="mui-component-select-country"
        className="MuiSelect-select MuiSelect-filled MuiFilledInput-input MuiInputBase-input css-d9oaum-MuiSelect-select-MuiInputBase-input-MuiFilledInput-input"
        id="mui-component-select-country"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="notranslate"
        >
          â€‹
        </span>
      </div>
      <input
        aria-hidden={true}
        autoFocus={true}
        className="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
        disabled={false}
        name="country"
        onAnimationStart={[Function]}
        onChange={[Function]}
        required={true}
        tabIndex={-1}
        value=""
      />
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconFilled css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
        data-testid="ArrowDropDownIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
    </div>
  </div>
</div>
`;
